Content.Tests/DMProject/Tests/Builtins/splicetext_runtime.dm

```diff
@@ -0,0 +1,7 @@
+// RUNTIME ERROR
+
+/proc/RunTest()
+    splicetext("banana", 12, -1, "test") //bad text or out of bounds
+    splicetext("banana", 3, -5, "test") //bad text or out of bounds
+    splicetext("banana", 0, 6, "laclav") //bad text or out of bounds
+    splicetext("abcdef", 4, 3, "test") //bad text or out of bounds
```

Comment:
This test will pass as long as any of these 4 lines cause an error. That means one of them could run without an error, and the test would still pass.

Maybe take advantage of the new try/catch support to instead fail if any one of these don't throw an exception?